crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -676,6 +671,350 @@ TEST_P(HMAC_ServiceIndicatorTest, HMACTest) {
   ASSERT_TRUE(check_test(hmacTestVector.expected_digest, mac.data(), mac_len, "HMAC KAT"));
 }
 
```

Comment:
Definitely need some failure paths with these tests: non-approved key lengths and non-approved hash algorithms